App.jsx

```diff
@@ -1,23 +1,59 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, FlatList, View } from 'react-native';
+import axios from 'axios';
+import Article from './components/Article';
+import Hero from './components/Hero';
 
 const App = () => {
+  const [articles, setArticles] = useState([]);
+  const fetchArticles = async () => {
+    const response = await axios.get(
```

Comment:
you can put it in a separate file, like on desktop clients

Reply:
![Screenshot 2021-05-26 at 11 29 29](https://user-images.githubusercontent.com/60035332/119637166-ab274400-be15-11eb-90d9-17834f34b0cd.png)

Reply:
;)

Reply:
xD

Reply:
wasn't there when I commented))